go.mod

```diff
@@ -1,10 +1,11 @@
-module github.com/openshift/prom-label-proxy
+module github.com/prometheus-community/prom-label-proxy
 
 go 1.13
 
 require (
 	github.com/go-openapi/runtime v0.19.15
 	github.com/go-openapi/strfmt v0.19.5
+	github.com/openshift/prom-label-proxy v0.1.0 // indirect
```

Comment:
there are probably .go files still importing from this.

Reply:
No think it was my local import that was wrong, should be fixed now, thanks!

Reply:
PTAL :)